static/js/compose.js

```diff
@@ -581,8 +581,7 @@ exports.handle_keydown = function (event) {
     if ((isBold || isItalic || isLink) && isCmdOrCtrl) {
         var range = textarea.range();
 
-        function add_markdown(markdown) {
-            var textarea = $("#compose-textarea");
+        function add_markdown(markdown, textarea) {
```

Comment:
Shadowing the `textarea` variable here feels a bit strange.  The inner function already has access to it through closure.

I would kind of go one way or the other here:
* Move add_markdown to its own top-level function.
* Take advantage of the closure variables to make the code concise.